app/models/conversation_account.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+# == Schema Information
+#
+# Table name: conversation_accounts
+#
+#  id                      :bigint(8)        not null, primary key
+#  account_id              :bigint(8)
+#  conversation_id         :bigint(8)
+#  participant_account_ids :bigint(8)        default([]), not null, is an Array
+#  status_ids              :bigint(8)        default([]), not null, is an Array
```

Comment:
Not sure about `participant_account_ids`, but afaik, `status_ids` makes sense because it's ordered and only used to track the latest status in a thread more efficiently when deleting statuses.
(Although it probably doesn't handle toots received out-of-order that well, hm)